tests/test_contracts.py

```diff
@@ -14,3 +15,21 @@ def test_contract_manager():
     assert manager.get_event_abi('TokenNetwork', 'ChannelOpened')
     with pytest.raises(KeyError):
         manager.get_event_abi('TokenNetwork', 'NonExistant')
+
+
+def test_deploy_multiple_tokens(add_and_register_token):
+    """Deploy and register $DEPLOY_TOKENS tokens in a TokenNetworksRegistry"""
+    DEPLOY_TOKENS = 10
+    token_list = [
+        (random.randint(100, 1000000),
```

Comment:
there should be a newline after `(`

Reply:
you guys and your fancy newlines :)

Reply:
hehe, we need a autoformatter, so much churn on formatting.